cmd/ci-op-configs-mirror/main.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/interrupts"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/config"
+)
+
+const (
+	prowgenConfigFile = ".config.prowgen"
+)
+
+type options struct {
+	dryRun         bool
+	configDir      string
+	destinationDir string
+	fromOrg        string
+	toOrg          string
+}
+
+func gatherOptions() options {
+	o := options{}
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+
+	fs.StringVar(&o.configDir, "config-path", "", "Path to directory containing ci-operator configurations")
+	fs.StringVar(&o.destinationDir, "destination-path", "", "Path of the destination of the generated ci-operator configurations")
```

Comment:
Why do we need separate input and destination? I would expect us to be confined inside `ci-operator/config`, both input and output. Is this really needed?

Reply:
Yea you are right. I'll address this as well